scripts/vscode.patch

```diff
@@ -864,6 +864,31 @@ index d8de423529..5953e920c3 100644
 +		Object.defineProperty(this._options, 'useSameOriginForRoot', {
 +			value: true,
 +		});
+diff --git a/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts b/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts
+index d4cd525812..6904db5518 100644
+--- a/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts
++++ b/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts
+@@ -30,8 +29,0 @@ export default () => `
+-				<div class="section recent">
+-					<h2 class="caption">${escape(localize('welcomePage.recent', "Recent"))}</h2>
+-					<ul class="list">
+-						<!-- Filled programmatically -->
+-						<li class="moreRecent"><a href="command:workbench.action.openRecent">${escape(localize('welcomePage.moreRecent', "More..."))}</a><span class="path detail if_shortcut" data-command="workbench.action.openRecent">(<span class="shortcut" data-command="workbench.action.openRecent"></span>)</span></li>
+-					</ul>
+-					<p class="none detail">${escape(localize('welcomePage.noRecentFolders', "No recent folders"))}</p>
+-				</div>
+@@ -41,6 +33,5 @@ export default () => `
+-						<li class="keybindingsReferenceLink"><a href="command:workbench.action.keybindingsReference">${escape(localize('welcomePage.keybindingsCheatsheet', "Printable keyboard cheatsheet"))}</a></li>
```

Comment:
Just leaving this here as a note not to merge yet; we're gonna provide the URLs (via `product.ts`) instead of replacing the `href`s.